scripts/ci-e2e.sh

```diff
@@ -184,12 +177,15 @@ main(){
 
     # Set common variables
     export DOCKER_BUILDKIT=1
+    export PROVIDER_ID_FORMAT=v2
```

Comment:
Just a note , Now by default we use v2 provider id format https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/config/manager/manager.yaml#L30, May be in future PR we can remove this explicitly setting as v2.